package.json

```diff
@@ -29,7 +29,8 @@
     "axios": "^1.7.9",
     "fast-xml-parser": "4.5.1",
     "loader-utils": "2.0.4",
-    "tar": "^6.2.1",
+    "nanoid": "^5.0.9",
```

Comment:
Why was nanoid added to the root package.json?

Reply:
I may be misunderstanding, but I think the original PR updated `nanoid` in the root, should I revert that one?

![Screenshot 2025-01-22 at 10 46 25 AM](https://github.com/user-attachments/assets/5f4b89f9-e2e3-4c48-b4d1-fe1f1a553eb3)